modules/deployments-ab-testing-lb.adoc

```diff
@@ -113,23 +113,36 @@ spec:
 ----
 
 [id="deployments-ab-testing-lb-web_{context}"]
-== Managing weights using the web console
+== Managing weights of an existing route using the web console
 
 .Procedure
 
-. Navigate to the Route details page (Applications/Routes).
+. Navigate to the *Networking* -> *Routes* page.
 
-. Select *Edit* from the Actions menu.
+. Click the Actions menu {kebab} next to the Route you want to edit and select *Edit Route*.
```

Comment:
s/Actions/*Actions*/ ?
Our guidelines mention using asterisks for console items such as menus, although a quick search finds that we are not consistent with this. So whatever is most consistent in this doc book makes sense to me.